lib/fetchContent.js

```diff
@@ -187,8 +194,29 @@ function fetchResText(url, options) {
         .then(res => res.text());
 }
 
+function cachedFetchResText(url, options) {
+  const safeUrl = url.replace(/apiKey=[^&]+/, 'apiKey=...');
```

Comment:
The need to remove traces of apiKey is worrying. IMO the key shouldn't be passed to this function ins the first place. Why not use ```safeUrl = `${SAPI_PATH}?apiKey=...``` in that case?
Similarly, the `delete` should happen on options rather than the new object, or even before it reaches this function.

Reply:
refactoring most of this into sapiV1CapiV2.
Done.